### PR TITLE
DevWs for Zooz (The Smartest House) containing containing Zooz ZSE43 Tilt | Shock XS Sensor

### PR DESCRIPTION
**Pull request by KevinTSH**

### CODE REVIEW

**Comment by KevinTSH** [comment]
This device requires a custom DTH because it has useful configuration parameters and there aren't any generic built-in DTHs that support the contact sensor and acceleration sensor capabilities

**Comment by KevinTSH** [comment]
This was submitted about a week ago so I'm just following up to see if anyone plans on reviewing it...  Thanks

File: devicetypes/zooz/zooz-zse43-tilt-shock-xs-sensor.src/zooz-zse43-tilt-shock-xs-sensor.groovy [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+			break
+		case vibrationOnly:
+			value = vibrationValue
+			break
+		default:
+			value = "${contactValue}${vibrationValue.capitalize()}"
+	}
+
+	if (device.currentValue("contactVibration") != value) {
+		sendEvent(name: "contactVibration", value: value, displayed: false)
```
Comment (greens) [inline devicetypes/zooz/zooz-zse43-tilt-shock-xs-sensor.src/zooz-zse43-tilt-shock-xs-sensor.groovy:321]: Is `contactVibration` just contact && acceleration?

File: devicetypes/zooz/zooz-zse43-tilt-shock-xs-sensor.src/zooz-zse43-tilt-shock-xs-sensor.groovy [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+	}
+}
+
+void zwaveEvent(physicalgraph.zwave.commands.versionv1.VersionReport cmd) {
+	logDebug "${cmd}"
+	sendEvent(name: "firmwareVersion", value: (cmd.applicationVersion + (cmd.applicationSubVersion / 100)))
```
Comment (greens) [inline devicetypes/zooz/zooz-zse43-tilt-shock-xs-sensor.src/zooz-zse43-tilt-shock-xs-sensor.groovy:327]: Are you intending for this to be used elsewhere later? Is this just for displaying to the user?

**Review by greens — APPROVED** [review]

**Pull request merged**